crates/worker/src/partition/snapshots/repository.rs

```diff
@@ -175,8 +175,8 @@ impl SnapshotRepository {
         skip_all,
         err,
         fields(
-            snapshot_id = ?snapshot.snapshot_id,
-            partition_id = ?snapshot.partition_id,
+            snapshot_id = %snapshot.snapshot_id,
+            partition_id = %snapshot.partition_id,
```

Comment:
A small note on span-level fields. When the span is at level "debug" and you decide to filter logging at "error", those fields will not be emitted to tracing subscribers. So this is fine as long as we are not logging anything higher than `debug` in this function. From what I see, we have `info` and `warn` and those will not include the snapshot_id nor the partition_id if `debug` level is not enabled at runtime.

Unfortunately, it's a common pitfall and we have it in many places in code and eventually we should address them. This is a nice chance to fix this one since you are already at it :)

Reply:
Oof, I had exactly the inverse mental model of span severity levels - thank you for the correction! Let's clean this up.